src/App.jsx

```diff
@@ -134,11 +134,12 @@ const AppWrapper = () => {
         userSettingsNoFallback,
     } = data
 
+    const appsModuleDelimiter = apiVersion >= 42 ? '/' : ':'
     const startModules = (data.apps.modules || []).map((module) => ({
         id:
             module.defaultAction.substr(0, 3) === '../'
                 ? module.name
-                : `app:${module.name}`,
+                : `apps${appsModuleDelimiter}${module.name}`,
```

Comment:
Right now this is `apps:` (plural) or `apps/` -- shouldn't it be `app:` (singular) for the < 42?

Maybe this approach:
```js
    const appsModulePrefix= apiVersion >= 42 ? 'apps/' : 'app:'
    const startModules = (data.apps.modules || []).map((module) => ({
        id:
            module.defaultAction.substr(0, 3) === '../'
                ? module.name
                : appsModulePrefix + module.name,

Reply:
thanks for the catch! for whatever reason, it worked with `apps:` in v41 when I tested, but I confirmed with Morten that it should yes be:
- `app:` for v41 and prior
- `apps/` for v42 and later